examples/generate_image_dataset/poses.csv

```diff
@@ -1,101 +1,101 @@
 scene_id,location_x,location_y,location_z,rotation_pitch,rotation_yaw,rotation_roll
```

Comment:
Let's truncate to 10 poses, since this is the default behavior we define in `generate_poses.py`